.travis.yml

```diff
@@ -3,4 +3,4 @@ before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.9.4
   - export PATH=$HOME/.yarn/bin:$PATH
 node_js:
-  - "10.11.0"
+  - "8.12"
```

Comment:
It was intentional.  We were originally forcing 10 to support iid-verify, but since 8.12's release, that's no longer a requirement.  This project can follow the same upgrade path as others.  Once 10 is the recommended version, we can go back to it.